lib/apiservers/engine/backends/commit_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backends
+
+import (
+	"archive/tar"
+	"bytes"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/docker/docker/api/types/backend"
+
+	"github.com/vmware/vic/lib/imagec"
+)
+
+func getMockReader() (io.ReadCloser, error) {
```

Comment:
I would pass a testing t here and would use that logging instead. fatal call in tests is kinda rough.